pkg/start/start.go

```diff
@@ -324,13 +324,13 @@ func watchHostAgentEvents(ctx context.Context, inst *store.Instance, haStdoutPat
 
 type watchHostAgentEventsTimeoutKey = struct{}
 
-// WithWatchHostAgentEventsTimeout sets the value of the timeout to use for
```

Comment:
Good catch. I propose to fix this in a separate PR and for the whole codebase.

Created #2496

Reply:
Agreed on separate PR; I have removed these changes from this commit.